packages/css/src/index.scss

```diff
@@ -23,7 +24,6 @@
 @import "./page-heading/page-heading";
 @import "./page-menu/page-menu";
 @import "./paragraph/paragraph";
-@import "./switch/switch";
```

Comment:
Why remove `Switch`? This line and various other files.

Reply:
Oh I see, this branch hasn’t been rebased on or merged with the main branch. Can you do that first?

Reply:
Ah not exactly, It was rebased but (the switch was) not published. So I got a whole lot of errors trying to continue on this branch. I reverted the rebase and continued. I will re-apply those commits.